pmm-app/docker-compose.yaml

```diff
@@ -2,7 +2,7 @@ version: "3"
 services:
   pmm-server:
     container_name: pmm-server
-    image: percona/pmm-server:2
+    image: perconalab/pmm-server-fb:PR-968-f04e404
```

Comment:
Let's not commit our local images anymore, and use local docker compose file for it

Reply:
Ok, I left a note on the PR, I will remove that commit once I can rebase with PMM-2.0 to get Grafana 7.1 packages, but yes I still had no need to commit that change of the docker image, thanks.